noether_filtering/src/mesh/bspline_reconstruction.cpp

```diff
@@ -140,7 +141,7 @@ bool BSplineReconstruction::filter(const pcl::PolygonMesh& mesh_in, pcl::Polygon
   }
 
   // fit B-spline boundary curve
-  std::shared_ptr<pcl::on_nurbs::FittingCurve2dASDM> curve_fit;
+  std::shared_ptr<pcl::on_nurbs::FittingCurve2dASDM> curve_fit = nullptr;
```

Comment:
Unless I'm mistaken, I don't think there is actually a difference here.

Reply:
I'm not sure but it doesn't hurt

Reply:
Just wasn't sure if there was something in that change that I didn't know about